cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -1,28 +1,592 @@
+// BEGIN enhancement for normalize-legacy-addon
+
+/*
+ * Corrects list images handled incorrectly in IE 7.
```

Comment:
So this comment comes directly from https://github.com/cfpb/normalize-legacy-addon/blob/master/normalize-legacy-addon.css#L163-L171
It's an enhancement here for the situation we had a few weeks back where the bullets were showing up in the secondary nav. It should be corrected in `normalize-legacy-addon` and then removed here. So okay to keep it for this PR.